src/panel/category/PoiItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import OpeningHour from 'src/components/OpeningHour';
+import OsmSchedule from 'src/adapters/osm_schedule';
+import ReviewScore from 'src/components/ReviewScore';
+import poiSubClass from 'src/mapbox/poi_subclass';
+import PoiTitleImage from 'src/panel/poi/PoiTitleImage';
+
+const PoiItem = ({ poi }) => {
+  const reviews = poi.blocksByType.grades;
+  const address = poi.address || {};
+
+  const Subclass = () =>
+    poi.subClassName
+      ? <p className="u-text--subtitle u-firstCap">{poiSubClass(poi.subClassName)}</p>
+      : <br/>
```

Comment:
I don't understand why you need `<br />`. Let's aim for the minimal DOM and return nothing if we can. Or do we really want weird white spaces?

Reply:
IIRC, I found it a bit weird to get the address at the same position, with the same font as the `subClassName` (type of poi). So I prefered to add a `<br />` tag to keep the address at the usual position. I removed these `br` tags, let's see if it's ok for all.

Reply:
Looks ok to me.